src/mono/netcore/sample/wasm/browser/Makefile

```diff
@@ -18,4 +18,4 @@ clean:
 	rm -rf bin
 
 run:
-	cd bin/$(CONFIG)/publish && python3 server.py
+	$(DOTNET) serve -o -d bin/$(CONFIG)/publish -p 8000
```

Comment:
We should check if `dotnet-serve` is installed (`dotnet tool list | grep botnet-serve`?). If not, then either tell the user how to install it, or go ahead and do it.

Reply:
Thanks, I opted to tell the user how to install rather for now.